src/Tabs/index.jsx

```diff
@@ -116,10 +118,18 @@ function Tabs({
         if (!moreTabHasNotification && overflowChild.props.notification) {
           moreTabHasNotification = true;
         }
+        function handleOnKeyPress(e, eventKey) {
```

Comment:
I think this function gets initialised multiple times because it's inside of a `.map` loop. Can you move it out somewhere above maybe? I think it makes sense to put it near `handleDropdownTabClick` function definition (because it uses that function) and also probably rename it to something like `handleDropdownKeyPress ` so we know that's it's only used for the dropdown.

Reply:
This function is required for `useMemo` dependencies. 
I moved this function outside of `map` and renamed. 
Thanks 👍